public/stylesheets/main.css

```diff
@@ -153,3 +153,24 @@ h1 {
     }
 
 }
+
+footer {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 1rem;
+}
+
+.powered-by-giphy {
+  font-size: 0.85em;
+  display: block;
```

Comment:
By "native", I meant "default", i.e. that `<div>` elements will display block on their own without additional property declarations. See http://learnlayout.com/display.html for more deets.